spec/tasks/demodata_spec.rb

```diff
@@ -71,6 +71,12 @@
         expect(case1.hearings.map(&:hearing_days).map(&:size)).to eql([3, 3, 3])
       end
 
+      it "adds unique sequential sittingDays to hearing days" do
+        sitting_days = case1.hearings.flat_map { |h| h.hearing_days.map { |d| d.sittingDay.to_date } }
+        expected_dates = (0..8).map { |i| "2019-10-23".to_date + i.days }
+        expect(sitting_days).to contain_exactly(*expected_dates)
+      end
+
```

Comment:
may be clearer to write the letters in the blocks as full words instead?

Reply:
Have tried out but because there are nine dates this starts to cause problems with length of line or length of block. And if used as a let block, as below, it needs to be move away from the test which is less clear. Its a trade off but hopefully not too unclear.
e.g. 
```
let(:expected_dates) { %w[2019-10-23 2019-10-24 2019-10-25 2019-10-26 2019-10-27 2019-10-28 2019-10-29 2019-10-30 2019-10-31].map(&:to_date) }
```

The error message when failing is pretty explicit:

```
    expected collection contained:  [Thu, 24 Oct 2019, Fri, 25 Oct 2019, Sat, 26 Oct 2019, Sun, 27 Oct 2019, Mon, 28 Oct 2019, Tue, 29 Oct 2019, Wed, 30 Oct 2019, Thu, 31 Oct 2019, Fri, 01 Nov 2019]
       actual collection contained:    [Wed, 23 Oct 2019, Thu, 24 Oct 2019, Fri, 25 Oct 2019, Sat, 26 Oct 2019, Sun, 27 Oct 2019, Mon, 28 Oct 2019, Tue, 29 Oct 2019, Wed, 30 Oct 2019, Thu, 31 Oct 2019]
       the missing elements were:      [Fri, 01 Nov 2019]
       the extra elements were:        [Wed, 23 Oct 2019]
```